scripts/linters/general_purpose_linter.py

```diff
@@ -424,7 +428,7 @@
         'excluded_dirs': ()
     },
     {
-        'regexp': re.compile(r'[^.|\w|\s]map\('),
+        'regexp': re.compile(r'[^.|\s]\(map\('),
```

Comment:
Can you please explain this changes?

Reply:
This check was not catching anything before, now I changed it to catch `map` if it found a pattern like `something(map())`)

Reply:
Can you add a link to the line which this failed to check? I just wanted to see!

Reply:
Sorry my mistake here, I don't why I change it, changed back to original, it is working fine